COMPILING.md

```diff
@@ -170,6 +170,15 @@ Follow these instructions:
       ```
       bash.exe -login
       ```
+      Please note that this might open a different shell instead, especially if
+      you have installed other Linux subsystems previously. In the Cygwin 
+      shell, the Windows file system can be accessed via `/cygdrive`. If the
```

Comment:
:grey_question: I'm not sure I understand the purpose of this sentence: "In the Cygwin shell, the Windows file system can be accessed via `/cygdrive`". Is it relevant for finding which version of bash you're using?

Reply:
Thanks - much clearer!